targets/zephyr/Makefile.zephyr

```diff
@@ -116,21 +117,22 @@ endif
 	cmake -B$(INTERM) -H./ \
 	 -DENABLE_LTO=OFF \
 	 -DENABLE_VALGRIND=OFF \
+	 -DFEATURE_PROFILE=minimal \
 	 -DCMAKE_BUILD_TYPE=Release \
 	 -DCMAKE_VERBOSE_MAKEFILE=$(V) \
+	 -DMEM_HEAP_SIZE_KB=$(JERRYHEAP) \
 	 -DEXTERNAL_CMAKE_C_COMPILER=$(CC) \
 	 -DEXTERNAL_CMAKE_C_COMPILER_ID=GNU \
 	 -DEXTERNAL_CMAKE_SYSTEM_PROCESSOR=$(CPU) \
-	 -DEXTERNAL_MEM_HEAP_SIZE_KB=$(JERRYHEAP) \
 	 -DEXTERNAL_COMPILE_FLAGS="$(EXT_CFLAGS)" \
-	 -DEXTERNAL_CMAKE_SYSTEM_PROCESSOR=lakemont \
 	 -DEXTERNAL_LIBC_INTERFACE="$(ZEPHYR_LIBC_INC)" \
-	 -DCMAKE_TOOLCHAIN_FILE=build/configs/toolchain_external.cmake \
-	 -DEXTERNAL_BUILD_ENTRY_FILE=$(TARGET_ZEPHYR_SRC_DIR)/jerry-entry.c
+	 -DCMAKE_TOOLCHAIN_FILE=cmake/toolchain_external.cmake \
+	 -DEXTERNAL_BUILD_ENTRY_FILE=$(TARGET_ZEPHYR_SRC_DIR)/jerry-entry.c \
```

Comment:
There is no EXTERNAL_BUILD_ENTRY_FILE option too. If you want to use an external entry file, you need to turn off the JERRY_CMDLINE option, and create your target using the specified source files, set its properties, add include directories to the target, and specify libraries or flags.